_pages/install/frontend/3x/building-the-spartacus-storefront-from-libraries-3-4.md

```diff
@@ -76,7 +76,7 @@ When you run this command, you are asked to choose which features you would like
 - To install the latest "next" release, you can add `@next` at the end of the command.
 - To install the latest release candidate, you can add `@rc` at the end of the command.
 - To verify what versions of Spartacus libraries were installed, open `package.json` and look for `@spartacus`.
-- To bypass the schematics prompts and install Spartacus with a predefined set of features, you can use the `--interactive=false` flag. For more information, see [Adding Spartacus Core Libraries and Features to Your Angular Project]({{ site.baseurl }}/schematics/#adding-spartacus-core-libraries-and-features-to-your-angular-project).
+- To bypass the schematics prompts and install Spartacus with a predefined set of features, you can use the `--no-interactive` flag. For more information, see [Adding Spartacus Core Libraries and Features to Your Angular Project]({{ site.baseurl }}/schematics/#adding-spartacus-core-libraries-and-features-to-your-angular-project).
```

Comment:
Revert changes in this file. It's doc for an older version of Spartacus.

Although it is not in one of the archived doc site version, the page says it's for spartacus 3.4